src/conf.c

```diff
@@ -135,8 +135,10 @@ static int validate_common(cfg_t *cfg, const char *provider, int custom)
 	}
 
 	if (!custom && !cfg_getstr(cfg, "username")) {
-		cfg_error(cfg, "Missing username setting for DDNS provider %s", provider);
-		return -1;
+		if (strcmp(provider, "default@freemyip.com") != 0) {	
```

Comment:
My actual name is:   Tomasz 'Cadence' Grabowski
Do I need to create a new PR for that? I'm not quite sure how that works once PR was merged.